jobs/ruby/middle-ruby-developer.md

```diff
@@ -4,7 +4,7 @@ We're looking for a pro-active middle-level web developer with experience in Rub
 
 ## About the job
 
-- Salary 𝚫: up to 4500USD, gross per month.
+- Salary 𝚫: 1000-3000USD, gross per month.
```

Comment:
Can we leave the same style as in jun Ruby: up to 3000 USD, gross per month

Reply:
Done